src/main/java/com/sinthoras/visualprospecting/gui/xaeromap/waypoints/WaypointWithDimension.java

```diff
@@ -0,0 +1,28 @@
+package com.sinthoras.visualprospecting.gui.xaeromap.waypoints;
+
+import xaero.common.minimap.waypoints.Waypoint;
+
+public class WaypointWithDimension extends Waypoint {
+
+	private final int dimID;
+	private int currentDim;
+
+	public WaypointWithDimension(int x, int y, int z, String name, String symbol, int color, int dimID) {
```

Comment:
I prefer to identify the coordinate system of XYZ in their name: blockX, chunkX, pixelX. That helps you sometimes. Your call

Reply:
I think it's clear enough here that it's blocks, since how else would a waypoint be measured